### PR TITLE
feat: added playlist feature

### PR DESCRIPTION
**Pull request by paarthbajaj**
playlist

### CODE REVIEW

File: src/backend/utils/serviceUtil.js [hunk 1]
```diff
@@ -31,6 +31,58 @@ export const addToPlaylist = async (playlistName) => {
...
+      {
+        video: videoObj,
+      },
+      {
+        headers: {
+          authorization: encodedToken,
+        },
+      }
+    );
+    console.log(data);
```
Comment (t007rushi) [inline src/backend/utils/serviceUtil.js:50]: remove consoles

File: src/backend/utils/serviceUtil.js [hunk 2]
```diff
@@ -31,6 +31,58 @@ export const addToPlaylist = async (playlistName) => {
...
+    // } catch (err) {
+    //   console.log(err);
+    // }
+    // return playlistData;
```
Comment (t007rushi) [inline src/backend/utils/serviceUtil.js:54]: remove unwanted comments

File: src/backend/utils/serviceUtil.js [hunk 3]
```diff
@@ -31,6 +31,58 @@ export const addToPlaylist = async (playlistName) => {
...
+  }
+};
+export const getSelectedPlaylistVideos = async (selectedPlaylistId) => {
+  try {
+    const data = await axios.get(`/api/user/playlists/${selectedPlaylistId}`, {
+      headers: {
+        authorization: encodedToken,
+      },
+    });
+    console.log(data);
```
Comment (t007rushi) [inline src/backend/utils/serviceUtil.js:85]: same here

**Review by t007rushi — APPROVED** [review]
remove unwanted comments @ many places
else go ahead

**Review by ajit-sharmaGH — APPROVED** [review]
Everything else is fine

**Pull request merged**